client/a8c-for-agencies/sections/marketplace/pressable-overview/plan-selection/style.scss

```diff
@@ -60,6 +60,12 @@
 	&.regular-ownership {
 		padding-block-start: 32px;
 	}
+
+	&.refer-mode {
```

Comment:
Nitpicking: To adhere to the Calypso standard for CSS modifiers, we should format it with `is-*` for both `regular-ownership` and `refer-mode`.

Reply:
Thanks for the review, @jkguidaven!

Agree. Fixed it here: https://github.com/Automattic/wp-calypso/pull/93311/commits/6f1842fb2808c529797587564554a67bcc651992